sbc_compassion/models/correspondence_template.py

```diff
@@ -135,15 +135,15 @@ def get_gmc_layouts(self):
     def _get_default_checkboxes(self):
         return [
             (0, False, {'language_id': self.env.ref(
-                'child_compassion.lang_compassion_french').id}),
+                'child_switzerland.lang_compassion_french').id}),
```

Comment:
In fact, the module `sbc_compassion` has no access to module `child_switzerland`, because `child_switzerland` is a child module in the dependency list.

Therefore, you should remove this _default_checkboxes from here (and remove the call to the function in the field definition), and move it in the module `sbc_switzerland`, while you should add `child_switzerland` in the dependencies of the module `sbc_switzerland`.